hudi-client/src/main/java/org/apache/hudi/table/CleanExecutor.java

```diff
@@ -60,17 +59,17 @@
  * <p>
  * TODO: Should all cleaning be done based on {@link HoodieCommitMetadata}
  */
-public class HoodieCleanHelper<T extends HoodieRecordPayload<T>> implements Serializable {
+public class CleanExecutor<T extends HoodieRecordPayload<T>> implements Serializable {
```

Comment:
This looks like there should be an Executor<> interface if we are going to move to *executor style of naming ? Do you have plans to introduce one later ?

Reply:
Agree.. Renamed this to stay consistent with `RollbackExecutor`.. but may be we should rename that to `RollbackHelper`.  Will take a pass and rename both consistently.